changelogs/fragments/vault_syml_allow.yml

```diff
@@ -0,0 +1,2 @@
+bugfixes:
+  - vault secrets file now executes in the correct context when it is a symlink (not resolved to canonical file).
```

Comment:
Is that trailing period necessary when the sentence is not capitalized?

Reply:
idk